test/environmentCreate/SpecificationCreate.test.tsx

```diff
@@ -30,7 +30,7 @@ describe("<SpecificationCreate />", () => {
   });
 
   it("should switch the view to the yaml editor", () => {
-    const switchButton = component.getByLabelText("Switch to YAML Editor");
+    const switchButton = component.getByLabelText("YAML", { exact: false });
```

Comment:
curious does the `exact: false` acts almost like a regex where it searches for a string that contains the given substring (i.e. `YAML` here)

Reply:
That's my understanding

Reply:
thought so, but it was worth double checking